Dockerfile

```diff
@@ -1,23 +1,18 @@
-ARG BASE_IMAGE=docker.io/fedora:32
+FROM quay.io/ansible/python-builder:latest as builder
+# =============================================================================
 
-FROM ${BASE_IMAGE}
+ARG ZUUL_SIBLINGS=""
+COPY . /tmp/src
+RUN assemble
 
+FROM docker.io/fedora:32
+# =============================================================================
 # Install system packages for use in all images
-RUN dnf install -y \
-    python3-pip \
-    gcc \
-    rsync \
-    openssh-clients \
-    sshpass \
-    glibc-langpack-en \
-    git \
-    https://github.com/krallin/tini/releases/download/v0.19.0/tini_0.19.0-amd64.rpm && \
-    rm -rf /var/cache/dnf
```

Comment:
I'm totally behind this, but we're going to need to get it tested fully, and we don't yet have automation for it.

I'm a little skeptical of the resultant image not having pip.

I'm also skeptical off the bat seeing tini being removed. I tried removing it earlier and it was very wrong.

Reply:
I only drop tini, since I couldn't find an official rpm package or python package. So, I have no idea the difference between dumb-init and it.  I don't have a preference except, I don't feel using random package from web is a good idea

piip will be installed, that will come from python-base, which is not published yet. I hope to get that on monday

Reply:
Oh, as for testing. I've created https://github.com/ansible/ansible-builder/pull/130 in ansible-runner. My goal is to get integration tests running in zuul, then we can use depends-on on a ansible-builder PR, to reference this PR, and zuul will do the work to use the speculative container for quay.io/ansible/ansible-runner:devel.

Once we confirm ansible-builder, and any another project is great, we can merge with confidence